lib/Backends/BackendUtils.cpp

```diff
@@ -27,6 +27,10 @@ uint8_t *glow::collectConstants(
 
   // At compile time condense constants to a single block of memory.
   // This allows the graph to go away after compile time.
+  // If there are no constants return nullptr.
+  if (constantMaxSize == 0) {
```

Comment:
please, check all the usages for constants_ from symbol table and make sure we handle `nullptr` properly :)